### ISSUE TITLE
Idea to make Lutris easier to use for those who are inexperienced with Linux (Dropping .exe file in Lutris window to automatically make an entry)

### ISSUE THREAD

**Issue by Kyuunex** [lutris/agora#58]
The following is based on my experience helping a Windows user who is a gamer, transition to Linux and me having to teach them how to use Lutris. Despite my efforts, he ended up going back, which is fine, but I feel like, there may have been some stuff that could have been in place to make this experience as simple as possible, and me not having to write a few page tutorials for him to teach him how to use Lutris. One of them being what I am proposing right now. So, what I am proposing is not something, I, somewhat experienced Linux user would complain about not having for myself, but I am saying if it existed, those who are not experienced could very well benefit from it. Onto the feature request then...

I want a feature that pops up a window when a user drops an .exe file into the Lutris window. The window will contain 2 things, first, a field a user can type a name for that game. The second thing would be asking if the dropped file is the game's executable or an installer.

If it's a game executable, a game entry will be made, into the game list, the name will be what user entered, the runner will be set to Wine. In Game options, the executable path will be auto set to the path of that .exe the user dropped. Wine prefix will be set to the Default installation folder, followed by the game name or whatever.

If it's a game installer, a game entry will be made, into the game list, the name will be what user entered, the runner will be set to Wine. Wine prefix will be set to the Default installation folder, followed by the game name or whatever. The exe file will be run automatically in that prefix and the user will have to follow the installation to it's end. After the installation is done, it could technically auto set the the executable path based on the start menu shortcut that the installer created. (Lutris can filter out any shortcut that has 'uninstall' in the name and anything that leads to a web url or a help page. It can have something in place that picks the shortcut that leads to just the .exe of the game). Either that or ask the user to manually find the .exe.

Why do I think this would be a great addition? Because not every game has a written Lutris install script. 
+ For example I personally have lots of CDs/DVDs of old, obscure games from the 2000s. 
+ Many obscure, untranslated visual novels that come on CDs/DVDs also don't really have a Lutris install script. 
+ Some obscure games from developer's websites also don't have a Lutris install script. 
+ And lastly, some DRM free installers, from example GOG that the user already downloaded.
  + Although in case of GOG, Lutris servers can technically store the SHA-1/MD5/whatever hashes of games and the Lutris app can look them up automatically when a user drops the .exe, and find an already made install script like that.

(Could also do the same for non exe files, for Linux stuff, idk, but Windows stuff is a good start)

**Comment by ghost** [comment]
Yeah,that's a really cool idea,especially for installers,cause right now I install games in my default wine folder,then add it to Lutris and move all the files in the prefix that I created in Lutris and that takes too long and some installers won't work with my systems Wine version,so it would also be cool,if the .exe is an installer,to be able to choose which Wine version installs it.

**Comment by ghost** [comment]
>...The window will contain 2 things, first, a field a user can type a name for that game...

A thing I would add is,that the name will automatically be the name of the .exe,so if the name is correct you don't have to add it yourself and if the name isn't what you want,that you can just make that field empty and enter a name you want.

**Comment by ghost** [comment]
Another thing I would add is,that you don't have to drop the .exe (or linux executable),but you also can just open the .exe with Lutris in a file manager or something and then the window you mentioned pops up.

**Comment by ghost** [comment]
Another cool feature would be if it would work for steam windows games mod installers,so I don't have to use protontricks.

**Comment by ghost** [comment]
@Kyuunex I tried Gamehub and with that adding games is easier,cause you can select if you want to add an installed game or install a windows game directly in a prefix.
The problem is that even if it has those cool features,it doesn't have some important features,like changing between many wine versions and add and enable env vars easily for all games.
That's the problem with Linux gaming,that things like game launchers aren't so good on Linux right now.
Hopefully Lutris or Gamehub will become faster and easier to use and will have more features in the future,cause then more people could switch to Linux without any problem.

**Comment by qnxt** [comment]
one other feature that i think would be nice if Lutris had would be if it could check what stuff the game executable file imports and use that during the prefix setup, 

example: it would check the imports and if it sees mfplat.dll, it would set up media foundation stuff for that
if it sees, msvcp140.dll, it would set up vcrun2015
and so on

one dirty way to see what stuff the executable imports is using the strings tool
```
$ strings '/mnt/ti/games/SteamLibrary/steamapps/common/Little Busters! English Edition/LITBUS_WIN32.exe' | grep -i .dll
qDll
SetDefaultDllDirectories
KERNEL32.dll
USER32.dll
GDI32.dll
ADVAPI32.dll
SHELL32.dll
ole32.dll
DSOUND.dll
WINMM.dll
IMM32.dll
d3d11.dll
dxgi.dll
steam_api.dll
AVRT.dll
MFPlat.DLL
```

**Comment by soredake** [comment]
https://github.com/bottlesdevs/Bottles

**Comment by ghost** [comment]
@soredake Thanks,that's much better than this.
